sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/SpecifyStringComparison.cs

```diff
@@ -35,7 +44,10 @@ void ValidCalls()
             "".StartsWith("", StringComparison.CurrentCulture); // Compliant
             "".StartsWith("", true, CultureInfo.CurrentCulture); // Compliant - CultureInfo implies string formatting
 
+            MyMethod1("", "");
             MyMethod2("");
+
+            Contains("", new[] { "" }, StringComparer.OrdinalIgnoreCase); // Compliant
```

Comment:
also add Noncompliant test

Reply:
I will let it like this here, as this is the real case that related to the issue (The `Contains` methods here have the exact same signatures as the ones defined by the framework).
I'll add another similar method that does have a `Noncompliant` variant as well